### PR TITLE
ATO-7: Set up vpc and deploy stacks

### PR DESCRIPTION
**Pull request by CarlyG55**
## What
Setting up stacks for int and prod deployments

## Checklist

- [ ] Impact on orch and auth mutual dependencies has been checked.

- [ ] A UCD review has been performed.

### CODE REVIEW

File: ci/stack-orchestration/configuration/integration/integration-orch-be-pipeline/parameters.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "ParameterKey": "RequireManualApproval",
+    "ParameterValue": "No"
+  },
+  {
+    "ParameterKey": "SAMStackName",
+    "ParameterValue": "integration-orch-be-deploy"
+  },
+  {
+    "ParameterKey": "SigningProfileArn",
+    "ParameterValue": "arn:aws:signer:eu-west-2:767397776536:/signing-profiles/SigningProfile_Lne9jMArpvlM"
```
Comment (isaac-GDS) [inline ci/stack-orchestration/configuration/integration/integration-orch-be-pipeline/parameters.json:64]: This references the signer we use in `build` . Do we need to create a new signer and reference that ?

File: ci/stack-orchestration/configuration/production/production-orch-be-pipeline/parameters.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    "ParameterKey": "RequireManualApproval",
+    "ParameterValue": "No"
+  },
+  {
+    "ParameterKey": "SAMStackName",
+    "ParameterValue": "production-orch-be-deploy"
+  },
+  {
+    "ParameterKey": "SigningProfileArn",
+    "ParameterValue": "arn:aws:signer:eu-west-2:767397776536:/signing-profiles/SigningProfile_Lne9jMArpvlM"
```
Comment (isaac-GDS) [inline ci/stack-orchestration/configuration/production/production-orch-be-pipeline/parameters.json:64]: Same comment as integration pipeline

File: ci/stack-orchestration/configuration/integration/integration-orch-be-pipeline/parameters.json (hunk 1 above)
Comment (CarlyG55) [reply ci/stack-orchestration/configuration/integration/integration-orch-be-pipeline/parameters.json:64]: [nope](https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3059908609/How+to+deploy+a+SAM+application+with+secure+pipelines#:~:text=Typically%20AWS%20Signer%20only%20deploys%20in%20the%20build%20account%20and%20the%20outputs%20SigningProfileArn%20and%20SigningProfileVersionArn%20are%20referenced%20in%20the%20other%20accounts%20that%20require%20it.) :)

Comment (isaac-GDS) [reply ci/stack-orchestration/configuration/integration/integration-orch-be-pipeline/parameters.json:64]: Cool - thanks.

**Review by isaac-GDS — APPROVED** [review]

**Review by ethanmills — DISMISSED** [review]

**Review by ethanmills — DISMISSED** [review]

**Review by ethanmills — APPROVED** [review]

**Pull request merged**